RenderStatePackagerLib/interface/DeviceObjectDescriptorParser.h

```diff
@@ -0,0 +1,305 @@
+/*
```

Comment:
After another thought I suggest renaming this interface to IRenderStateMarkupParser and to name our json file format diligent render state markup (DRSM).